extensions/jaxb/deployment/src/main/java/io/quarkus/jaxb/deployment/JaxbProcessor.java

```diff
@@ -170,9 +177,25 @@ void processAnnotationsAndIndexFiles(
         JAXB_ANNOTATIONS.stream()
                 .map(Class::getName)
                 .forEach(className -> {
-                    proxyDefinitions.produce(new NativeImageProxyDefinitionBuildItem(className, Locatable.class.getName()));
+                    proxyDefinitions.produce(new NativeImageProxyDefinitionBuildItem(className,
+                            "org.glassfish.jaxb.core.v2.model.annotation.Locatable"));
                     addReflectiveClass(reflectiveClass, true, false, className);
                 });
+        proxyDefinitions
+                .produce(new NativeImageProxyDefinitionBuildItem("org.glassfish.jaxb.core.marshaller.CharacterEscapeHandler"));
+        proxyDefinitions.produce(new NativeImageProxyDefinitionBuildItem("org.glassfish.jaxb.characterEscapeHandler"));
```

Comment:
Is this a class? I only found [org.glassfish.jaxb.core.marshaller.CharacterEscapeHandler](https://github.com/eclipse-ee4j/jaxb-ri/blob/master/jaxb-ri/core/src/main/java/org/glassfish/jaxb/core/marshaller/CharacterEscapeHandler.java)